src/sentry/static/sentry/app/components/events/interfaces/crashHeader.jsx

```diff
@@ -166,3 +171,13 @@ class CrashHeader extends React.Component {
 }
 
 export default CrashHeader;
+
+const StyledTitleInfo = styled('div')(({theme}) => ({
```

Comment:
Can we use the template literal style?

Reply:
yes sure 😉 but please, what are the reasons that led you to prefer template literals instead object?